sa/storage-authority.go

```diff
@@ -31,6 +33,114 @@ func digest256(data []byte) []byte {
 	return d.Sum(nil)
 }
 
+var dialectMap map[string]interface{} = map[string]interface{}{
+	"sqlite3":  gorp.SqliteDialect{},
+	"mysql":    gorp.MySQLDialect{Engine: "InnoDB", Encoding: "UTF8"},
+	"postgres": gorp.PostgresDialect{},
+}
+
+// Models
+type Registration struct {
```

Comment:
I believe all of the types in this file only need to be accessed within the sa package, right? So we should name them lowercase to be unexported. Might also want to add model at the end, e.g. registrationModel, since it's likely the types will overlap with a variable name if they are lowercased.